docker-compose.yml

```diff
@@ -32,5 +32,30 @@ services:
       dockerfile: verdict/Dockerfile
     networks:
       - maratona-net
+  postgres:
+    ports:
+      - "5432:5432"
+    image: "postgres:alpine"
+    environment:
+        POSTGRES_PASSWORD: password
```

Comment:
Should we care that we are using a password password?

Reply:
I think this isn't an issue for now since we're still using a local database.